### DISCUSSION TITLE
new sensor VL53L5CX

### DISCUSSION THREAD

**Discussion by serkanMzlm** [bitcraze/discussions#386] [Q&A] [answered]
Can 3D mapping be done using the VL53L5CX sensor?

I am using this sensor and I have created a Multiranger like build where it uses bitcraze. In this structure, there are 4 VL53L5CX sensors to see the sides.
These sensors can be 8x8 or optionally 4x4, thus speeding up the HZ.

**Comment by victorhook** [comment, marked as answer]
Short answer: Yes it is!

This sensor is really cool and I know that ETH Zurich has done some cool stuff with it, eg check out [here](https://arxiv.org/pdf/2208.12624.pdf), and [here](https://www.youtube.com/watch?v=FyipTqjBGuM). I think they have mainly done obstacle avoidance though, but perhaps there's some information that's interesting for you.

I also built a deck with 5 of the VL53L5CX sensors this year when doing I was doing my [thesis](https://lup.lub.lu.se/student-papers/search/publication/9090270). I did some basic 3D point cloud plotting with the 8x8 distance matrix. However, I did not do any proper _mapping_, but it can definitely be done, here's the [repo](https://github.com/victorhook/slamdeck) for my deck if you're curious.

**Reply by serkanMzlm** [reply]
Thanks
